pkg/package/toml_test.go

```diff
@@ -77,6 +77,8 @@ func TestUnMarshalTOML(t *testing.T) {
 	assert.Equal(t, modfile.Pkg.Name, "MyKcl")
 	assert.Equal(t, modfile.Pkg.Edition, "v0.0.1")
 	assert.Equal(t, modfile.Pkg.Version, "v0.0.1")
+	assert.Equal(t, modfile.Pkg.Include, []string{"src/", "README.md", "LICENSE"})
```

Comment:
Test failed.

Reply:
the test is passing on my machine 
![image](https://github.com/kcl-lang/kpm/assets/87874775/ae4842a8-8f6f-4e87-aa20-b963f1a13a21)

Reply:
OK. I will merge this PR first, and investigate it now.